test.e2e/spec/services.spec.js

```diff
@@ -223,6 +223,29 @@ define(['angular', 'given', 'util'], function(angular, given, util) {
         });
       });
 
+      it('can create multiple new resources', function() {
+        var arr = MyModel.createMany([
+          { name: 'new' },
+          { name: 'newest'},
+          { name: 'last'}
+        ], 
+        function() {
+          expect(arr).to.have.property('length', 3);
+        });
+        return arr.$promise.then(function() {
+          var found = MyModel.find(
+            {},
+            function() {
+              expect(found).to.have.property('length', 4);
+              found.forEach(function(obj){
+                expect(obj).to.have.property('name', obj.property);
```

Comment:
I find this confusing. What is `obj.property` and where is it coming from? Also the fact that the assertion has to take into account records created by previous tests is not great.

I am proposing to modify the test as follows:

```js
      var arr = MyModel.createMany([
         { name: 'one', multi: true },
         { name: 'two', multi: true },
         { name: 'three', multi: true }
       ], 
       function() {
         expect(arr).to.have.property('length', 3);
       });

      return arr.$promise.then(function() {
         var found = MyModel.find(
           { filter: { where: { multi: true } } },
           function() {
             var namesFound = found.map(function(it) { return it.name; });
             expect(namesFound).to.eql(['one', 'two', 'three']);
           });
```

The "multi" property is used to filter only the records created by this test. Asserting on the array of names has the benefit of a more helpful assertion error being reported.

Thoughts?